src/main/java/Chat.java

```diff
@@ -0,0 +1,127 @@
+import java.util.*;
+
+public class Chat {
+
+    /**
+     * This Chat version is just supposed to take in the commands,
+     * and return them, thats all. Mitran.Di.CHatri. Ton. Udd. Gayi. Laundi. ah.
+     */
+
+    public static String LINE_BREAK = "---------------";
+    public static String GREETING = "FUNNY FELLA WEIIIIII! \nHow can I help Mr Singhhhhh?";
+    public static String BYE = LINE_BREAK + "\n Byeeeee, come back again ah!\n" +
+            LINE_BREAK;
+    public static String LINE_INTRO = "Nah, here's your list";
+    public static String MARK_MESSAGE = "Power la Mr Bosssssss, mark alr bro!";
+    public static String UNMARK_MESSAGE = "No probs bro, unmarked already!";
+
+    public static void main(String[] args) {
+
+        // an array of tasks
+        Task[] list = new Task[100];
+        int counter = 0;
+        System.out.println(GREETING);
+        // new Data Structure to hold the tasks
+        ArrayList<Task> tasks = new ArrayList<Task>();
+
+        // takes in the incoming prompt
+        Scanner sc = new Scanner(System.in);
```

Comment:
Perhaps give this scanner a lengthier name other than sc? So anyone reading the code below will not get confused what sc is...maybe call it scanner instead which is clearer?